api/cloud/oc_cloud_store_internal.h

```diff
@@ -28,6 +28,22 @@
 extern "C" {
 #endif
 
+typedef struct oc_cloud_store_t
+{
+  oc_string_t ci_server;     ///< URL of OCF Cloud.
+  oc_string_t auth_provider; ///< The name of the Authorisation Provider through
+                             // which access token was obtained.
+  oc_string_t uid;
```

Comment:
@jkralik any idea what is this `uid` property? It's not mentioned in specification, and it is only used by cloud resource GET/POST handlers, otherwise it's not used anywhere in code.

Reply:
It is needed for non-jwt token. It determines uid of cloud user that can manipulate with device.
It is specified in https://openconnectivity.org/specs/OCF_Cloud_Security_Specification.pdf page 15. (table 4)